Jenkinsfile

```diff
@@ -12,10 +13,11 @@ pipeline {
     */
     cron(BRANCH_NAME == "master" ? "H H(4-6) * * *" : "")
   }
-  environment { PATH="${tool 'docker-latest'}/bin:$PATH" }
+  // environment { PATH="${tool 'docker-latest'}/bin:$PATH" }
```

Comment:
❔ What is the purpose of commenting this out? Not sure what it was doing. Should it be deleted?

Reply:
i think this was bringing in docker to be used for building and pushing the image. I added the shared library and started docker by calling dockerd in the Build Images stage. It can actually be eliminated and not commented out

Reply:
i removed the commented line